Orange-OpenSource/Orange-Boosted-Bootstrap
# Prepare release v5.0.0

chore(merge main): patched commit → bf09367

  - Firefox ESR
  - IE11 (v4 only)
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
  - if bumping a minor or major version:
    - [x] Add docs version to `_data/docs-versions.yml`
    - [x] Manually change `docs_version` in `_config.yml`
    - [x] Update redirects in docs frontmatter
    - [x] Move docs to the new docs' subfolder?
    - [x] Ensure Algolia indexes new release content ([probably requires a PR](https://github.com/algolia/docsearch-configs/pull/1954))
  - check wrong matches in `CHANGELOG.md`, and maybe `site/content/docs/<version>/migration.md`
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - and probably maintain [a ship list (eg for v4.4.0)](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/226)
- [ ] merge (on `v5-dev` for now)
  - attach zip file
  - paste CHANGELOG / Ship list in the release's description
- [ ] run `build/ship.sh x.x.x` to…
  - Pull the latest changes update new version numbers
  - Compile CSS and JS
  - Compress dist files
  - Generate the hashes and update our config's cdn section
  - Compile the docs
  - Copy-pasta built docs to `bs-docs`-named clone of Boosted.
  - [ ] copy `../bs-docs` to the `gh-pages` branch
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
